docs/schema_mapping.py

```diff
@@ -317,6 +317,13 @@
         'Pydantic standard "format" extension',
         'IPv4 or IPv6 network as used in ``ipaddress`` module',
     ],
+    [
+        'StrictBool',
```

Comment:
I'm not sure this is right, I think the schema for `StrictBool` should be the same as standard `bool`

Reply:
Not sure I follow? This is identical to `bool` apart from the name of it?

Reply:
yes, sorry.